### ISSUE TITLE
Strange NullReference Exception 

### ISSUE THREAD

**Issue by Kubsy** [MontyTRC89/Tomb-Editor#631]
This exception seems to appear randomly and it is quite hard to trace (imo)

![image](https://user-images.githubusercontent.com/80340234/202870932-9034932d-9ac6-4f1e-a5c0-65a45c28a2ea.png)

```
Application does not support just-in-time (JIT)
debugging. See the end of this message for details.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at TombEditor.Gizmo.get_Position()
   at TombLib.Graphics.BaseGizmo.Draw(Matrix4x4 viewProjection)
   at TombEditor.Controls.PanelRendering3D.OnDraw()
   at TombLib.Controls.RenderingPanel.OnPaint(PaintEventArgs e)
   at System.Windows.Forms.Control.PaintWithErrorHandling(PaintEventArgs e, Int16 layer)
   at System.Windows.Forms.Control.WmPaint(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
```

**Comment by Lwmte** [comment]
Possibly fixed in 84d5278d4a8cd6b021c5a44e4122e7d63d5bda78

**Comment by Lwmte** [comment]
Not reported in latest versions, most likely not happening anymore.
